esmvaltool/recipes/recipe_ecs_scatter_cmip6.yml

```
@@ -0,0 +1,303 @@
+#          - {dataset: IPSL-CM6A-LR, grid: gr, ensemble: r9i1p1f1}
+#          - {dataset: CNRM-CM6-1, grid: gr, ensemble: r1i1p1f2}
+#          - {dataset: CNRM-ESM2-1, grid: gr, ensemble: r1i1p1f2}
+#          - {dataset: MIROC6, grid: gn, ensemble: r1i1p1f1}
+      hur:
+        preprocessor: pp500
+        reference_dataset: ERA-Interim
+        start_year: 2001
+        end_year: 2012
+        exp: historical
+        mip: Amon
+        project: CMIP6
+        additional_datasets:
+          - {dataset: ERA-Interim, project: OBS, type: reanaly, version: 1, tier: 3}
+          - {dataset: GISS-E2-1-G, grid: gn, ensemble: r1i1p1f1}
+#          - {dataset: IPSL-CM6A-LR, grid: gr, ensemble: r9i1p1f1}
+#          - {dataset: CNRM-CM6-1, grid: gr, ensemble: r1i1p1f2}
+#          - {dataset: CNRM-ESM2-1, grid: gr, ensemble: r1i1p1f2}
+#          - {dataset: MIROC6, grid: gn, ensemble: r1i1p1f1}
+      rsut:
+        reference_dataset: CERES-EBAF
+        start_year: 2001
+        end_year: 2012
+        exp: historical
+        mip: Amon
+        project: CMIP6
+        additional_datasets:
+          - {dataset: CERES-EBAF, project: obs4mips, level: L3B, version: Ed2-7, tier: 1}
+          - {dataset: GISS-E2-1-G, grid: gn, ensemble: r1i1p1f1}
+#          - {dataset: IPSL-CM6A-LR, grid: gr, ensemble: r9i1p1f1}
+#          - {dataset: CNRM-CM6-1, grid: gr, ensemble: r1i1p1f2}
+#          - {dataset: CNRM-ESM2-1, grid: gr, ensemble: r1i1p1f2}
+#          - {dataset: MIROC6, grid: gn, ensemble: r1i1p1f1}
+      rsutcs:
+        reference_dataset: CERES-EBAF
+        start_year: 2001
+        end_year: 2012
+        exp: historical
+        mip: Amon
+        project: CMIP6
+        additional_datasets:
+          - {dataset: CERES-EBAF, project: obs4mips, level: L3B, version: Ed2-7, tier: 1}
+          - {dataset: GISS-E2-1-G, grid: gn, ensemble: r1i1p1f1}
+#          - {dataset: IPSL-CM6A-LR, grid: gr, ensemble: r9i1p1f1}
+#          - {dataset: CNRM-CM6-1, grid: gr, ensemble: r1i1p1f2}
+#          - {dataset: CNRM-ESM2-1, grid: gr, ensemble: r1i1p1f2}
+#          - {dataset: MIROC6, grid: gn, ensemble: r1i1p1f1}
+      rsdt:
+        reference_dataset: CERES-EBAF
+        start_year: 2001
+        end_year: 2012
+        exp: historical
+        mip: Amon
+        project: CMIP6
+        additional_datasets:
+          - {dataset: CERES-EBAF, project: obs4mips, level: L3B, version: Ed2-7, tier: 1}
+          - {dataset: GISS-E2-1-G, grid: gn, ensemble: r1i1p1f1}
+#          - {dataset: IPSL-CM6A-LR, grid: gr, ensemble: r9i1p1f1}
+#          - {dataset: CNRM-CM6-1, grid: gr, ensemble: r1i1p1f2}
+#          - {dataset: CNRM-ESM2-1, grid: gr, ensemble: r1i1p1f2}
+#          - {dataset: MIROC6, grid: gn, ensemble: r1i1p1f1}
+    scripts:
+      ecs_scatter:
+        diag: covrefl
+        calcmm: True
+        ecs_file: $diag_scripts/emergent_constraints/ecs_cmip.nc
+        legend_outside: False
```

```suggestion
        legend_outside: false
```
Please use `yamllint` to ensure the recipes are according to the standards

---

All yamllint warnings have been addressed.